### PR TITLE
Bugfix/tabs

### PR DESCRIPTION
**Pull request by dumbbillyhardy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (GustavoFernandes) [inline .gitignore]: Unnecessary `.gitignore`

File: main.js [hunk 2]
```diff
@@ -3,54 +3,55 @@ function split() {
...
+
 	for (var i = 0; i < arr.length; i++) {
 		var line = arr[i].trim();
-		
+
+		line = line.replace(/\s+/g, " "); 
 		var dollarIndex = line.indexOf('$');
 		if (itemCost == null && dollarIndex > -1) {
-			itemAmount = Number(line.substring(0, line.indexOf(' ')));
+			itemAmount = Number(line.substring(0, line.indexOf(" ")));
```
Comment (GustavoFernandes) [inline main.js]: Keep single quotes

**Review by GustavoFernandes — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (billy-hardy) [inline .gitignore]: should I add build tasks to make it necessary then?

File: main.js (hunk 2 above)
Comment (billy-hardy) [inline main.js]: fin

**Review by GustavoFernandes — APPROVED** [review]

**Pull request merged**